app/models/utils/bulk_media_packager.rb

```diff
@@ -27,25 +22,13 @@ def media_objects_scope
     end
 
     def media_meta
-      {space_on_disk: space_on_disk?, media_size: bytes_to_mb(media_size)}
-    end
-
-    def media_size
-      @media_size ||= media_objects_scope.sum("active_storage_blobs.byte_size")
-    end
-
-    def space_on_disk?
-      stat = Sys::Filesystem.stat("/")
-      # need to leave space for images, zip file, and copy of zip file while attaching to operation
-      space_left = bytes_to_mib(stat.block_size * stat.blocks_available) -
-        bytes_to_mib(media_size * 2)
-      space_left >= DISK_ALLOWANCE
+      {space_on_disk: space_on_disk?, media_size: bytes_to_mb(download_size)}
     end
 
     def download_and_zip_images
       FileUtils.mkdir_p(Rails.root.join(TMP_DIR))
 
-      media_ids = media_objects_scope.pluck("media_objects.id")
+      media_ids = attachment_scope.pluck("media_objects.id")
```

Comment:
Spec failure: I think this is supposed to be `download_scope` now.

Reply:
fixed bulk media failing specs